waiter/src/waiter/service_description.clj

```diff
@@ -837,8 +837,14 @@
   "Loads the service description for the specified service-id including any overrides."
   [kv-store service-id service-description-defaults metric-group-mappings &
    {:keys [effective? nil-on-missing? refresh] :or {effective? true nil-on-missing? true refresh false}}]
```

Comment:
Nitpick: Why is this parameter called `refresh` and not `refresh?`? Both of the other optional flags have the `?` suffix.

Reply:
I'd prefer to make the rename refactor in a separate PR, there are other instances of `refresh` in the codebase that will also need to be fixed.

Reply:
`refresh` no longer is a parameter to this function.